resources/crates.py

```diff
@@ -0,0 +1,9 @@
+from pulumi_command import local
+
+
+def add_owner(username: str):
+        add_owner_command = local.Command(
+        f"add_owner_{username}",
+        create=f"cargo owner --add {username}",
```

Comment:
I am looking this [doc page](https://doc.rust-lang.org/cargo/commands/cargo-owner.html). 
It has the following

`
cargo owner [options] --add login [crate]
`

Shouldn't we add `[crate]`? Without it, it operates on the crate in local folder. But this folder won't have any.

Reply:
Good catch, lemme fix.

Reply:
Fixed here: f1831ac